File: AshishChopra/authSignUp/WebApplication5/WebApplication5/Models/SignUp.cs
using System;
using System.Collections.Generic;

namespace WebApplication5.Models
{
    public partial class SignUp
    {
        public string Name { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string ConfirmPassword { get; set; }
        public string CollegeName { get; set; }
        public string CollegeId { get; set; }
    }
}

File: AshishRawat/Oauth/Oauth/signup.js
$(document).ready(function () {
    $("#submit").click(function (e) {
        e.preventDefault();
        var email = document.getElementById("Email1").value;
        var name = document.getElementById("Name").value;

        var collegename = document.getElementById("Collegename").value;

        var collegeid = parseInt(document.getElementById("Collegeid").value);

        var pass = document.getElementById("Password").value;
        console.log(name + ' ' + email + ' ' + collegename);
        $.ajax("http://localhost:56355/api/values", {
            type: "POST",
            dataType: "json",
            contentType: "application/json",
            // crosDomain:"true",
            data: JSON.stringify(
                {
                    "name": name,
                    "username": email,
                    "collegeid": collegeid,
                    "collegename": collegename,
                    "password": pass
                }
            ),
            success: function (data, status) {
                console.log("hello", data, status);
                window.location.replace("login.html");
            },
            error: function () {
                alert("Something went wrong")
            }
        });

    });
});
